### DISCUSSION TITLE
Discord invite broken?

### DISCUSSION THREAD

**Discussion by Charuru** [nocodb/nocodb#5352] [Q&A] [answered]
No discord invite links work anymore, could someone post a new one?

**Comment by wingkwong** [comment, marked as answer]
Seems not broken. Which link are you using?

The link I used is https://discord.gg/5RgZmkW

![image](https://user-images.githubusercontent.com/35857179/227200956-1253aa6c-70ed-4201-af9a-b155cac422e4.png)

**Reply by Charuru** [reply]
I used it too and it failed for me, said the invite was expired or invalid or something, but I might've done something and now it's fine. Not sure what happened, might've been on my end. I also realize that I had already joined the discord months ago and forgot, whoops.
